flink-table/flink-sql-parser/src/main/codegen/data/Parser.tdd

```diff
@@ -224,7 +224,6 @@
     "RESUME"
     "TABLES"
     "TIMESTAMP_LTZ"
-    "TRY_CAST"
```

Comment:
I am curious why TRY_CAST is removed as a keyword. I had assumed it would still be a keyword but Calcite would now provide the implementation.

Reply:
because it was added to keywords on Calcite level, no need to add it one more time